src/content/docs/new-relic-control/agent-control/overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Agent Control overview"
+metaDescription: "Agent Control is an agent supervisor that can be used to deploy, update, and manage New Relic agents."
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+Agent Control is an agent supervisor that can be used to deploy, update, and manage New Relic agents. 
+
+## Prerequisites
+Before you begin deploying Agent Control to your Kubernetes clusters, ensure that you meet the following prerequisites:
+
+- **Helm 3:** Helm version 3 must be installed on your machine. For installation instructions, see [Installing Helm](https://helm.sh/docs/intro/install).
+- **New Relic license key:** You'll need a New Relic license key to report telemetry to your New Relic account.
+- **Kubernetes cluster name:** Have the name of your Kubernetes cluster ready, as it will be referenced during the installation process.
+- **Auth Domain Manager:** To set up Agent Control and connect it to Fleet Control, you must have the Authentication Domain Manager role.
+- **Existing instrumentation:** If you already have a Kubernetes cluster instrumented with New Relic, you have to uninstall the instrumentation and then re-install it:
```

Comment:
I created a PR adding a page to do a small deep dive on this https://github.com/newrelic/docs-website/pull/20049